types/basic_auth.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) OpenFaaS Project. All rights reserved.
```

Comment:
This should be `OpenFaaS Author(s)` unless it had a different copyright in its original source file?